templates/corosync.conf.erb

```diff
@@ -166,3 +166,9 @@ nodelist {
 }
 <% end -%>
 <% end -%>
+
+<% unless @watchdog_device.nil? -%>
```

Comment:
If you write it like this, there is no diff for existing users when they don't use a watchdog device:
```suggestion
<% unless @watchdog_device.nil? -%>

```

I'm not sure if that diff shows up well, but the idea is to include the empty line inside the block.